libcloud/compute/drivers/dimensiondata.py

```diff
@@ -166,8 +171,8 @@ def destroy_node(self, node):
             'server/deleteServer',
             method='POST',
             data=ET.tostring(request_elm)).object
-        result = findtext(body, 'responseCode', TYPES_URN)
-        return result == 'IN_PROGRESS'
+        responseCode = findtext(body, 'responseCode', TYPES_URN)
+        return responseCode == 'IN_PROGRESS' or responseCode == 'OK'
```

Comment:
Minor style thing - you could also do `return responseCode in ['IN_PROGRESS', 'OK']`

Also, for consistency, please use _ notation for variable names (`response_code`) :)